quilter.py

```diff
@@ -0,0 +1,360 @@
+        for dir_name, subdir_list, file_list in os.walk(extract_folder):
+            for fname in file_list:
+                if str.endswith(fname, raster_exts):
+                    vrt_list.append(os.path.join(dir_name, fname))
+
+        vrt_opts = gdal.BuildVRTOptions(resampleAlg='cubic')
+        vrt = gdal.BuildVRT(vrt_path, vrt_list, options=vrt_opts)
+        vrt = None  # releases file handle
+
+        if compress:
+            co=["bigtiff=yes", "compress=lzw", "tiled=yes"]
+        else:
+            co=["tiled=yes"]
+
+        # overwrite creation options for jpeg compression of aerial imagery
+        if len(vrt_list) > 0:
+            ds = gdal.Open(vrt_list[0], gdal.GA_ReadOnly)
+            md = ds.GetMetadata('IMAGE_STRUCTURE')
+            if 'COMPRESSION' in md and 'JPEG' in md["COMPRESSION"]:
+                co = ["bigtiff=yes", "compress=jpeg", "tiled=yes"]
+                gdal.SetConfigOption('COMPRESS_OVERVIEW', 'JPEG')
+                if 'YCbCr' in md['COMPRESSION']:
+                    co.append("photometric=ycbcr")
+                    gdal.SetConfigOption('PHOTOMETRIC_OVERVIEW', 'YCBCR')
+
+        # project if desired
+        if project:
+            print("\nProjecting {} to {}".format(raster_outpath, projection))
+            warp_opts = gdal.WarpOptions(dstSRS=projection, 
+                                         resampleAlg="cubic", 
+                                         format="GTiff", 
+                                         multithread=True,
+                                         creationOptions=co,
+                                         callback=gdal_progress_callback)
+            ds = gdal.Warp(raster_outpath, vrt_path, options=warp_opts)
+            ds = None  # releases file handle
+
+            print("\nBuilding overviews...")
+            dso = gdal.Open(raster_outpath, gdal.GA_ReadOnly)
+            dso.BuildOverviews("NEAREST", [2, 4, 8, 16], gdal_progress_callback)
+            dso = None
+        
+        # otherwise, just translate vrt to tif
+        else:
+            print("\nMerging into {}".format(raster_outpath))
+            trans_opts = gdal.TranslateOptions(format="GTiff", 
+                                               creationOptions=co, 
+                                               callback=gdal_progress_callback)
+            ds = gdal.Translate(raster_outpath, vrt_path, options=trans_opts)
+            ds=None
+
+            print("\nBuilding overviews...")
+            dso = gdal.Open(raster_outpath, gdal.GA_ReadOnly)
+            dso.BuildOverviews("NEAREST", [2, 4, 8, 16], gdal_progress_callback)
+            dso = None
+    
+except ImportError as e:
+    print("\n===========\nDON'T PANIC\n===========")
+    if "gdal" in e.args[0]:
+        print("Can't import GDAL python bindings. Recommend installing latest GDAL from the conda-forge channel via conda: conda install -c conda-forge gdal.")
+    print("\nPython error message:")
```

Comment:
another style nit, we typically use single quotes for python except for when you have the `'` in the string.

Reply:
Yeah... I'm kinda all over the place with that. I'll be more cautious to use `'`.